cypress/integration/issue-list.spec.ts

```diff
@@ -73,13 +73,12 @@ describe("Issue List", () => {
       cy.get("tbody tr:first").contains(mockIssues2.items[0].message);
     });
 
-    //TODO this tests fails on the pre-hood "beforeEach"
-    // it("persists page after reload", () => {
-    //   cy.get("@next-button").click();
-    //   cy.contains("Page 2 of 3");
+    it("persists page after reload", () => {
```

Comment:
I've seen this test failing randomly as well. Randomly failing tests are called flaky. If it fails in the CI you can simply re-run the job:

![image](https://user-images.githubusercontent.com/9722792/187850830-2e51e2a3-2656-4d67-a748-bc2f3ff0a2d0.png)

Reply:
In regards to this specifically, I added the cypress videos to .gitignore file and it seems to have made it less flaky.